### PR TITLE
Editable cells

### PR DESCRIPTION
**Pull request by worksofliam**
Introducing the `update` prefix. When used in combination with a `select` statement that only references one table, then the cells will become updatable. A fully qualified table must be used.

* Try using `update: select * from schema.table`
* Click on a cell for it to become editable
* Either click off the cell or use Enter to submit
* The update is ran as soon as the cell loses focus

Tested column types:

- Numbers
- Strings/Chars
- Date/Times/Timestamp
- Can set as null

### CODE REVIEW

File: src/views/results/html.ts [hunk 1]
```diff
@@ -76,6 +94,131 @@ export function generateScroller(basicSelect: string, isCL: boolean, withCancel?
...
+              else if (target.tagName.toLowerCase() == "td") {
+                // get the inner div
+                chosenValue = target.firstChild.innerText;
+                editableNode = target;
+                trWithColumn = target;
+              }
+
+              if (trWithColumn && trWithColumn.column) {
+                const chosenColumn = trWithColumn.column;
+                const chosenColumnDetail = updateTable.columns.find(col => col.name === chosenColumn);
```
[the remark this answers is not in the document]
Comment (worksofliam) [reply src/views/results/html.ts:122]: Good idea

File: src/views/results/html.ts [hunk 2]
```diff
@@ -76,6 +94,131 @@ export function generateScroller(basicSelect: string, isCL: boolean, withCancel?
...
+                  }
+                }
+
+                // Already editable, just return
+                if (editableNode.contentEditable === 'true') return;
+                editableNode.contentEditable = true;
+                editableNode.focus();
+
+                const keydownEvent = (e) => {
+                  if (e.keyCode === 13) {
```
[the remark this answers is not in the document]
Comment (worksofliam) [reply src/views/results/html.ts:141]: Need to test this to confirm.

File: src/views/results/html.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (worksofliam) [reply src/views/results/html.ts:122]: I added this line below. Not replaced.

**Review by forstie — APPROVED** [review]
After loads of testing and a few fixes from Liam, this looks great.

**Pull request merged**